src/test/kotlin/no/nav/familie/ef/sak/arbeidsforhold/ArbeidsforholdControllerTest.kt

```diff
@@ -0,0 +1,51 @@
+package no.nav.familie.ef.sak.no.nav.familie.ef.sak.arbeidsforhold
+
+
+import io.mockk.verify
+import no.nav.familie.ef.sak.OppslagSpringRunnerTest
+import no.nav.familie.ef.sak.amelding.AMeldingInntektDto
+import no.nav.familie.ef.sak.arbeidsforhold.ArbeidsforholdDto
+import no.nav.familie.ef.sak.arbeidsforhold.ekstern.ArbeidsforholdClient
+import no.nav.familie.ef.sak.fagsak.FagsakRepository
+import no.nav.familie.ef.sak.repository.fagsak
+import no.nav.familie.ef.sak.repository.fagsakpersoner
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.familie.kontrakter.felles.getDataOrThrow
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.mockito.Mock
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.mock.mockito.MockBean
+import org.springframework.boot.test.web.client.exchange
+import org.springframework.http.HttpEntity
+import org.springframework.http.HttpMethod
+import org.springframework.http.ResponseEntity
+import java.time.LocalDate
+import java.util.UUID
+
+internal class ArbeidsforholdControllerTest : OppslagSpringRunnerTest() {
+
+    @Autowired private lateinit var fagsakRepository: FagsakRepository
+    @Autowired private lateinit var arbeidsforholdClient: ArbeidsforholdClient
+
+    private val fagsak = fagsak(identer = fagsakpersoner(setOf("1")))
+
+    @BeforeEach
+    fun setUp() {
+        headers.setBearerAuth(lokalTestToken)
+        fagsakRepository.insert(fagsak)
+    }
+
+    @Test
+    internal fun `skal hente arbeidsforhold med ansettelsesperiodeFom i query param`() {
+        val arbeidsforhold = hentArbeidsforhold(fagsak.id)
+        assertThat(arbeidsforhold.body!!.getDataOrThrow()).isNotNull
+        verify { arbeidsforholdClient.hentArbeidsforhold("1", LocalDate.of(2021, 1, 1)) }
```

Comment:
Noen som skjønner hvorfor denne feiler med: `Verification failed: call 2 of 2: OAuth2AccessTokenResponse(child of #1#36).getAccessToken()) was not called` ?
Hvor er det det blir definert at det forventes kall mot getAccessToken?

Reply:
Mocken er ikke i bruk 😁 
så må mocken også endres til `fun arbeidsforholdClient(): ArbeidsforholdClient {`

Reply:
ah, det var der magien lå! Takk! :D